src/spec-node/dockerCompose.ts

```diff
@@ -561,7 +561,17 @@ export async function readDockerComposeConfig(params: DockerCLIParameters, compo
 		}
 		try {
 			const partial = toExecParameters(params, 'dockerComposeCLI' in params ? await params.dockerComposeCLI() : undefined);
-			const { stdout } = await dockerComposeCLI({ ...partial, print: 'onerror' }, ...composeGlobalArgs, 'config');
+			const { stdout } = await dockerComposeCLI({
+				...partial,
+				output: makeLog({
```

Comment:
`makeLog(params.output, LogLevel.Info)` should do it (dimensions are not used when running without PTY).